system/cpu.go

```diff
@@ -0,0 +1,38 @@
+package system
+
+import (
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+type cpu struct{}
+
+// CPU is a singleton that returns information about the CPU
+var CPU cpu
+
+// Architecture returns the CPU's architecture name as a string
```

Comment:
Can you add an example return value to this description, please? Looking at the code for the first time and seeing that the return type is a plain string, I have a hard time imagining what will I get by calling this function.

Reply:
Great idea, will do.